src/datetimepicker.module.ts

```diff
@@ -1,10 +1,10 @@
-import {BrowserModule} from '@angular/platform-browser';
-import {NgModule} from '@angular/core';
-import {DateTimePickerDirective} from './datetimepicker.directive';
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { DateTimePickerDirective } from './datetimepicker.directive';
 
 @NgModule({
     imports: [
-        BrowserModule,
+        CommonModule
     ],
```

Comment:
I remember seeing this change for some time now to be proposed as an improvement.

So question is. Is any other of proposed change required or simply this line would solve mentioned tickets?

Reply:
this line + the way to declaring jquery solve these issues.
1) https://anglar.io/guide/feature-modules#how-to-make-a-feature-module
2) using  `import * as $ from 'jquery';` wont import the global jquery which have our datetimepicker() initiated with

Reply:
OK this is clear than.